clevertap-core/src/main/java/com/clevertap/android/sdk/inapp/evaluation/TriggersMatcher.kt

```diff
@@ -50,7 +50,7 @@ class TriggersMatcher {
      */
     @VisibleForTesting
     internal fun match(trigger: TriggerAdapter, event: EventAdapter): Boolean {
-        if (event.eventName != trigger.eventName) {
+        if (event.eventName != trigger.eventName && (event.profileAttrName == null || !event.profileAttrName.equals(trigger.profileAttrName, true))) {
```

Comment:
need tests for this case

Reply:
Will add and update unit tests for all new changes